DataConnectors/Connector_syslog_WatchGuardFirebox.json

```diff
@@ -0,0 +1,117 @@
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **<enter the Kusto Function alias>** in queries and workbooks. [Follow steps to get this Kusto Function>](<Link to Kusto Function on Azure Sentinel GitHub>)  ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Watchguard",
+            "baseQuery": "Firebox"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Firebox log category count in last 24 hours",
+            "query": "Firebox\n | where HostName contains 'WatchGuard'\n| where TimeGenerated >= ago(24h)\n| summarize count() by SeverityLevel\n"
+        },
+        {
+            "description" : "Firebox top 10 messages in last 24 hours",
+            "query": "Firebox\n | where HostName contains 'WatchGuard'\n| where TimeGenerated >= ago(24h)\n| summarize count() by SyslogMessage\n| top 10 by count_ desc"
+        },		
+		{
+            "description" : "Firebox top 10 HostIP in last 24 hours",
+            "query": "Firebox\n | where HostName contains 'WatchGuard'\n| where TimeGenerated >= ago(24h)\n| summarize count() by HostIP\n| top 10 by count_ desc"
+        }	
+    ],
+    "dataTypes": [
+        {
+            "name": "Syslog(Firebox)",
+            "lastDataReceivedQuery": "Firebox\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "Firebox\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "write permission is required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "delete": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "", 
+            "description": ">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](Link to Kusto Function on Azure Sentinel GitHub) to create the Kusto Functions alias, **enter the Kusto Function alias**",

```

Comment:
Again, this should have the function alias defined and has a valid link to the function.